roles/contrail_common/tasks/hosts_to_ip.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- set_fact:
+    contrail_masters_ip: ""
```

Comment:
This var should be defined in `roles/contrail_common/defaults/mail.yml`

Reply:
will resolve this on my next commit